services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationServiceITest.kt

```diff
@@ -232,24 +272,42 @@ class ApplicationServiceITest : DatabaseTest() {
         verify { cableReportServiceAllu wasNot Called }
     }
 
+    @Test
+    fun `getAllApplicationsForUser returns applications for user hankkeet`() {
```

Comment:
I think this tests everything `getAllApplicationsForUser returns applications for the correct user` tests, so that one could be removed.

Reply:
It checked the cable report data so I didn't touch it.

Reply:
Oh yes, that's right.